incubator/sparkoperator/values.yaml

```diff
@@ -1,6 +1,10 @@
-operatorImageName: lightbend/sparkoperator
-operatorVersion: 2.3.1
+operatorImageName: gcr.io/spark-operator/spark-operator
+operatorVersion: v2.3.1-v1alpha1-latest
```

Comment:
please confirm that this is the version you want to make available.

Reply:
Yes it is for now. When Spark 2.4 is out in a few weeks, I'll make another PR to update the image to use it.